release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1011,6 +1024,54 @@ KMM includes the following features for connected environments only:
+[id="scalability-and-performance-talm-updates"]
+==== {cgu-operator-first}
+
+{cgu-operator-first} now provides more detailed status information and messages, and redesigned conditions. 
+You can use the `ClusterLabelSelector` field for greater flexibility in selecting clusters for update.
+You can use timeout settings to determine what happens if an update fails for a cluster, for example, skipping the failing cluster and continuing to upgrade other clusters, or stopping policy remediation for all clusters. 
+For more information see xref:../scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc[Topology Aware Lifecycle Manager for cluster updates].
+
+[id="scalability-and-performance-mount-namespace"]
+==== Mount namespace encapsulation (Technology Preview)
+
+Encapsulation is the process of moving all Kubernetes-specific mount points to an alternative namespace to reduce the visibility and performance impact of a large number of mount points in the default namespace. Previously, mount namespace encapsulation has been deployed transparently in {product-title} specifically for Distributed Units (DUs) installed using GitOps ZTP. In {product-title} v4.12, this functionality is now available as a configurable option.
+
+A standard host operating system uses systemd to constantly scan all mount namespaces: both the standard Linux mounts and the numerous mounts that Kubernetes uses to operate. The current implementation of Kubelet and CRI-O both use the top-level namespace for all container and Kubelet mount points. Encapsulating these container-specific mount points in a private namespace reduces systemd overhead and enhances CPU performance. Encapsulation can also improve security, by storing Kubernetes-specific mount points in a location safe from inspection by unprivileged users. 
+
+For more information, see xref:../scalability_and_performance/optimizing-cpu-usage.adoc#optimizing-cpu-usage[Optimizing CPU usage with mount namespace encapsulation].
+
+[id="ocp-4-12-ztp-sno-workload-partitioning"]
+==== Changing the workload partitioning CPU set in {sno} clusters that are deployed with GitOps ZTP
+
+You can configure the workload partitioning CPU set in {sno} clusters that you deploy with GitOps ZTP.
+To do this, you specify cluster management CPU resources with the `cpuset` field of the `SiteConfig` custom resource (CR) and the `reserved` field of the group `PolicyGenTemplate` CR.
+The value that you set for `cpuset` should match the value set in the cluster `PerformanceProfile` CR `.spec.cpu.reserved` field for workload partitioning.
+
+For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-using-hub-cluster-templates_ztp-advanced-policy-config[Using hub templates in PolicyGenTemplate CRs].
```

Comment:
For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc#ztp-sno-du-enabling-workload-partitioning_sno-configure-for-vdu[Workload partitioning].